snippets/modern.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Classic Holy Grail Layout
+tags: layout,beginner
+---
+
+Create a quick template  
+
+- Use `display: grid` generates a block-level grid.
+- Use `grid-template: auto 1fr auto / auto 1fr auto` = grid-template-rows/grid-template-columns.
+- Use `grid-column`  is the line where the item begins, the line where the item ends.
+
+```html
+<div class="parent">
+    <header>Header</header>
+    <div class="left-sidebar" >Left Sidebar</div>
+    <main>Content</main>
+    <div class="right-sidebar">Right Sidebar</div>
+    <footer>Footer</footer>
+</div>
+  
+```
+
+```css
+.parent {
+    display: grid;
+    height: 100vh;
+    grid-template: auto 1fr auto / auto 1fr auto;
+}
+  
+header {
+    background: lightpink;
```

Comment:
Unnecessary coloration should be avoided. Better not color the areas or use some other technique.